example.py

```diff
@@ -1124,6 +1181,51 @@ def core(self, slothy):
         slothy.optimize_loop("layer5678_start")
 
 
+class intt_dilithium_1234_5678(Example):
+    def __init__(self, var="", arch=AArch64_Neon, target=Target_CortexA72, timeout=None):
+        name = f"intt_dilithium_1234_5678"
+        infile = name
+
+        if var != "":
+            name += f"_{var}"
+            infile += f"_{var}"
+        name += f"_{target_label_dict[target]}"
+
+        super().__init__(infile, name, rename=True, arch=arch, target=target, timeout=timeout)
+
+    def core(self, slothy):
+        conf = slothy.config.copy()
+
+        slothy.config.reserved_regs = [
+            f"x{i}" for i in range(0, 6)] + ["x30", "sp"]
```

Comment:
@dop-amin Could we take the opportunity to mark `x18` as reserved for all [i]NTT examples?